package/src/store/OfflineDB.ts

```diff
@@ -0,0 +1,92 @@
+import { AbstractOfflineDB, StreamChat } from 'stream-chat';
+import type {
+  GetAppSettingsType,
+  GetChannelsForQueryType,
+  GetChannelsType,
+  GetLastSyncedAtType,
+  UpsertAppSettingsType,
+  UpsertUserSyncStatusType,
+} from 'stream-chat';
+
+import * as api from './apis';
+import { SqliteClient } from './SqliteClient';
+
+export class OfflineDB extends AbstractOfflineDB {
+  constructor({ client }: { client: StreamChat }) {
+    super({ client });
+  }
+
+  upsertCidsForQuery = api.upsertCidsForQuery;
+
+  upsertChannels = api.upsertChannels;
+
+  // FIXME
```

Comment:
Remove the comments if there is nothing left to be fixed here and everywhere else in this file. Or may be add a comment of what has to be fixed if this has to be re-iterated.